views/tasks/includes/legacy-relevant-legal-entity.html

```diff
@@ -0,0 +1,76 @@
+{% for psc in pscList %}
```

Comment:
Rather than 'legacy...' suggest naming this file 'single-relevant-legal-entity.html'. I don't think we've used 'legacy' elsewhere during out refactorings.